packages/ra-core/src/dataProvider/useGetList.ts

```diff
@@ -71,11 +71,18 @@ const useGetList = <RecordType = Record>(
         { type: 'getList', resource, payload: { pagination, sort, filter } },
         options,
         // data selector (may return [])
+        // We don't want the list to reset to first page after a bulk delete if the current page still exists
+        // But CrudDeleteMany reset all "cachedRequests" for this resource.
+        // It caused ids to be empty, and useListController to automatically reset to page 1
+        // Now, it "cachedRequests" is undefined, we try default ids before setting an empty ids array
```

Comment:
we already have such a system in `useListController`. I think that's where it belongs, not in `useGetList`.

Reply:
Fixed, and then removed because already fixed in master since I opened this PR ...